### PR TITLE
[#1 Kill Tasks functionality] Create task kill views

### PR DESCRIPTION
**Pull request by kennyt**
This is just the view part. Actions will come next

px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/components/KillTaskModal.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  }
+
+  getConfirmTextBody(selectedItems, selectedItemsLength) {
+    let {action} = this.props;
+    let secretText;
+
+    if (selectedItemsLength === 1) {
+      secretText = selectedItems[0].id;
+    } else {
+      secretText = 'these tasks';
```
Comment (jfurrow) [inline src/js/components/KillTaskModal.js]: Should we call this something besides `secretText`?

File: src/js/components/TaskView.js [hunk 2]
```diff
@@ -136,9 +168,48 @@ class TaskView extends mixin(SaveStateMixin, StoreMixin) {
...
+
+  getKillTaskModal(checkedItems, hasCheckedTasks) {
+    if (!hasCheckedTasks) {
+      return null;
+    }
+
+    let {killAction} = this.state;
+
+    let tasks = this.getCheckedItems(this.props.tasks);
+    return (
```
Comment (jfurrow) [inline src/js/components/TaskView.js]: #nit newline, maybe remove the newline between two `let` statements.

File: src/js/components/KillTaskModal.js (hunk 1 above)
Comment (kennyt) [inline src/js/components/KillTaskModal.js]: done. they're called `bodyText` now

File: src/js/components/TaskView.js (hunk 2 above)
Comment (kennyt) [inline src/js/components/TaskView.js]: done

**Pull request merged**